front/app/containers/CookiePolicy/index.tsx

```diff
@@ -51,10 +57,21 @@ const CookiePolicy = () => {
             <StyledContentContainer>
               <Fragment name="pages/cookie-policy/content">
                 <PageTitle>
-                  {formatMessage(messages.cookiePolicyTitle)}
+                  {pageAttributes?.title_multiloc ? (
+                    <T value={pageAttributes.title_multiloc} />
```

Comment:
To be sure: the title would just be text/not contain HTML, right?

Reply:
Yes!  

It’s a bit unintuitive, isn’t it, that content created in the Quill editor doesn’t render properly in a `QuillEditedContent` component? I haven’t checked how this works, just find it unintuitive. 🙃

Reply:
Ah, I see it was actually a problem at the level of the `T` component 👍